modular_skyrat/modules/modular_items/code/modular_glasses.dm

```diff
@@ -0,0 +1,297 @@
+  name = "diagnostic HUD aviators"
+  desc = "A heads-up display capable of analyzing the integrity and status of robotics and exosuits. This HUD has been fitted inside of a pair of sunglasses."
+  icon_state = "aviator_diagnostic"
+  off_state = "aviator"
+  toggleable = TRUE
+  vision_correction = FALSE
+  flash_protect = FLASH_PROTECTION_NONE
+  hud_type = DATA_HUD_DIAGNOSTIC_BASIC
+  hud_trait = TRAIT_DIAGNOSTIC_HUD
+  glass_colour_type = /datum/client_colour/glass_colour/lightorange
+
+// Science Aviators
+/obj/item/clothing/glasses/hud/aviator/science
+  name = "science aviators"
+  desc = "A pair of tacky purple aviator sunglasses that allow the wearer to recognize various chemical compounds with only a glance."
+  icon_state = "aviator_sci"
+  off_state = "aviator"
+  toggleable = TRUE
+  vision_correction = FALSE
+  flash_protect = FLASH_PROTECTION_NONE
+  glass_colour_type = /datum/client_colour/glass_colour/purple
+  resistance_flags = ACID_PROOF
+  armor = list(MELEE = 0, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 0, BIO = 0, FIRE = 80, ACID = 100)
+  clothing_traits = list(TRAIT_REAGENT_SCANNER, TRAIT_RESEARCH_SCANNER)
+
+/obj/item/clothing/glasses/hud/aviator/security/prescription
+  name = "prescription security HUD aviators"
+  desc = "A heads-up display that scans the humanoids in view and provides accurate data about their ID status and security records. This HUD has been fitted inside of a pair of sunglasses with toggleable electrochromatic tinting which. Has lenses that help correct eye sight."
+  vision_correction = TRUE
+/obj/item/clothing/glasses/hud/aviator/health/prescription
+  name = "prescription medical HUD aviators"
+  desc = "A heads-up display that scans the humanoids in view and provides accurate data about their health status. This HUD has been fitted inside of a pair of sunglasses which has lenses that help correct eye sight."
+  vision_correction = TRUE
+/obj/item/clothing/glasses/hud/aviator/meson/prescription
+  name = "prescription meson HUD aviators"
+  desc = "Used by engineering and mining staff to see basic structural and terrain layouts through walls, regardless of lighting conditions. This HUD has been fitted inside of a pair of sunglasses which has lenses that help correct eye sight."
+  vision_correction = TRUE
+/obj/item/clothing/glasses/hud/aviator/diagnostic/prescription
+  name = "prescription diagnostic HUD aviators"
+  desc = "A heads-up display capable of analyzing the integrity and status of robotics and exosuits. This HUD has been fitted inside of a pair of sunglasses which has lenses that help correct eye sight."
+  vision_correction = TRUE
+/obj/item/clothing/glasses/hud/aviator/science/prescription
+  name = "prescription science aviators"
+  desc = "A pair of tacky purple aviator sunglasses that allow the wearer to recognize various chemical compounds with only a glance, which has lenses that help correct eye sight."
+  vision_correction = TRUE
```

Comment:
please add spaces in-between your items, it'll make your code a lot more readable.
```suggestion
/obj/item/clothing/glasses/hud/aviator/security/prescription
  name = "prescription security HUD aviators"
  desc = "A heads-up display that scans the humanoids in view and provides accurate data about their ID status and security records. This HUD has been fitted inside of a pair of sunglasses with toggleable electrochromatic tinting which. Has lenses that help correct eye sight."
  vision_correction = TRUE
  
/obj/item/clothing/glasses/hud/aviator/health/prescription
  name = "prescription medical HUD aviators"
  desc = "A heads-up display that scans the humanoids in view and provides accurate data about their health status. This HUD has been fitted inside of a pair of sunglasses which has lenses that help correct eye sight."
  vision_correction = TRUE
  
/obj/item/clothing/glasses/hud/aviator/meson/prescription
  name = "prescription meson HUD aviators"
  desc = "Used by engineering and mining staff to see basic structural and terrain layouts through walls, regardless of lighting conditions. This HUD has been fitted inside of a pair of sunglasses which has lenses that help correct eye sight."
  vision_correction = TRUE
  
/obj/item/clothing/glasses/hud/aviator/diagnostic/prescription
  name = "prescription diagnostic HUD aviators"
  desc = "A heads-up display capable of analyzing the integrity and status of robotics and exosuits. This HUD has been fitted inside of a pair of sunglasses which has lenses that help correct eye sight."
  vision_correction = TRUE
  
/obj/item/clothing/glasses/hud/aviator/science/prescription
  name = "prescription science aviators"
  desc = "A pair of tacky purple aviator sunglasses that allow the wearer to recognize various chemical compounds with only a glance, which has lenses that help correct eye sight."
  vision_correction = TRUE
```